hacks/ops-pod

```diff
@@ -183,12 +182,12 @@ while [[ $(kubectl -n $namespace get pods | sed -n -r "s/^$name.*Running.*$/Runn
 
 # exec into pod (and chroot into node if a node was selected)
 if [[ ${node_chroot} -eq ${TRUE} ]]; then
-  kubectl -n $namespace exec -ti $name -- bash -c "rm -rf /host/root/dotfiles 1> /dev/null; \
+  kubectl -n $namespace exec -ti $name -- bash -c 'rm -rf /host/root/dotfiles 1> /dev/null; \
```

Comment:
Why don't we use the `/hacks/start_chrooted` script here?

Reply:
We synced with @petersutter .
We should leave the setup and clean-up for the on-demand scripts for another PR.
I will remove the `start_chrooted` as well.